src/sentry/static/sentry/app/views/settings/organizationFiltersAndSampling/rules/rule/conditions.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+import {DynamicSamplingCondition} from 'app/types/dynamicSampling';
+
+import {getOperatorLabel} from './utils';
+
+type Props = {
+  conditions: Array<DynamicSamplingCondition>;
+};
+
+function Conditions({conditions}: Props) {
+  function getConvertedValue(value: DynamicSamplingCondition['value']) {
+    if (Array.isArray(value)) {
+      return (
+        <Values>
+          {value.map((v, index) => (
+            <React.Fragment key={v}>
```

Comment:
Is it possible to have duplicate values?

Reply:
I don't think so